src/sxastarter/src/components/play-travel/Hero.tsx

```diff
@@ -0,0 +1,109 @@
+  useSitecoreContext,
+} from '@sitecore-jss/sitecore-jss-nextjs';
+import Image from 'next/future/image';
+import Arrow from '../../assets/images/arrow.svg';
+
+interface Fields {
+  'Background Image': ImageField;
+  Title: Field<string>;
+  Description: Field<string>;
+  Link: LinkField;
+  Picture: ImageField;
+}
+
+type HeroProps = {
+  params: { [key: string]: string };
+  fields: Fields;
+};
+
+type ButtonProps = {
+  field: LinkField;
+  showArrow?: boolean | string;
+};
+
+const ButtonComponent = (props: ButtonProps): JSX.Element => (
+  <Button
+    fontFamily="Inter"
+    background={'linear-gradient(135deg, #F9EC7D 0%, #F5B100 100%)'}
+    borderRadius="100px"
+    padding={'0 16px 0 24px'}
+    transition="0.25s all"
+    _hover={{
+      background: 'linear-gradient(135deg, #F9EC7D 0%, #F5B100 100%)',
+    }}
+    color="#0C111F"
+    className="font-semibold md:text-lg h-8 lg:h-16"
+  >
+    <JssLink field={props.field} />
+    <Flex
+      backgroundColor="#0C111F"
+      className="h-6 w-6 lg:h-8 lg:w-8 justify-center align-center rounded-2xl ml-3.5"
+    >
+      <Image style={{ display: 'inline' }} alt={props.field.value.text || ''} src={Arrow} />
+    </Flex>
+  </Button>
+);
+
+const HeroDefaultComponent = (props: HeroProps): JSX.Element => (
+  <div className={`component hero ${props?.params?.styles}`}>
+    <div className="component-content">
+      <span className="is-empty-hint">Hero</span>
+    </div>
+  </div>
+);
+
+export const Default = (props: HeroProps): JSX.Element => {
+  const id = props.params?.RenderingIdentifier;
+
+  if (props) {
+    const { sitecoreContext } = useSitecoreContext();
+    
+    return (
+      <Box
+        id={id}
+        backgroundImage={`url('${props?.fields['Background Image']?.value?.src}')`}
+        backgroundRepeat="no-repeat"
+        backgroundSize="cover"
+        className={`flex md:items-center pb-6 md:pb-0 items-end w-full min-h-325 lg:min-h-820 lg:px-8 md:px-20 lg:py-5 md:py-0 px-6 overflow-hidden relative ${props.params?.styles}`}
+      >
+        {sitecoreContext.pageEditing ? <JssImage className="absolute top-0 left-0 h-full" field={props.fields['Background Image']} /> : ''}
+        <div className="container flex items-center z-10 justify-between flex-col md:flex-row">
+          <Box
+            maxWidth="600px"
+            minWidth={{ base: 'unset', md: '50%' }}
+            paddingTop={{ base: '30px', md: '0' }}
+            className="font-lora tex-white flex-1 lg:order-2 md:order-none"
+          >
+            <Box
+              fontFamily="Lora"
+              lineHeight="120%"
+              className="text-white text-2xl md:text-4xl lg:text-6xl font-semibold"
+            >
+              <JssRichText field={props.fields?.Title} />
+            </Box>
+            <Box
+              fontFamily="Mulish"
+              lineHeight="200%"
+              marginBottom="25px"
+              className="text-white text-xs sm:text-xs text-white pt-4 md:text-base"
```

Comment:
![image](https://user-images.githubusercontent.com/16080426/214511035-785b5a7f-d1a5-4488-95f0-5628dd26c065.png)
you are repeating "text-white" 2 times